mac

```diff
@@ -115,7 +115,6 @@ tap "caskroom/cask"
 brew "universal-ctags", args: ["HEAD"]
 brew "git"
 # git customization
-git config --global merge.conflictstyle diff3
```

Comment:
This was causing errors, but I didn't try to figure out why because I don't think we should be doing this anyways. People have different preferences for this, so I think they should change it if they want.

Reply:
*sad face*